testing/libsmi/APKBUILD

```diff
@@ -0,0 +1,38 @@
+# Contributor: Mikolaj Chwalisz <chwalisz@tkn.tu-berlin.de>
+# Maintainer: Mikolaj Chwalisz <chwalisz@tkn.tu-berlin.de>
+pkgname=libsmi
+pkgver="0.5.0"
+pkgrel=0
+pkgdesc="Library to access SMI management information"
+url="https://www.ibr.cs.tu-bs.de/projects/libsmi/"
+arch="all"
+license="Copyright (c) 1999-2008 Frank Strauss, Technical University of Braunschweig."
```

Comment:
It is the same as Tcl. In Tcl package it is marked as TCL on wikipedia: BSD-style. Which one should I put in?